src/modules/addscore.js

```diff
@@ -0,0 +1,22 @@
+export default class NewScore {
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+}
+
+const addScore = async (user, score) => {
+//   const apiLink = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+//   const shortlink = `${apiLink}games/`;
+//   const myId = 'NAjmpgjlPEaGPDuhK9lg';
```

Comment:
I think it is a good idea to remove the commented code. It makes your project looks messy. :+1: :smiley: :smiley: